__tests__/components/AccountSelectorField.spec.ts

```diff
@@ -31,7 +31,7 @@ describe('components/AccountSelectorField', () => {
                     currentAccount: null,
                 },
             );
-            const component = wrapper.vm as AccountSelectorField;
+            const component = wrapper.vm as any;
```

Comment:
why any?

Reply:
Okay, found a workaround, imported the `AccountSelectorFieldTs` file additionally.